lib/portlayer/event/collector/vsphere/collector.go

```diff
@@ -169,12 +169,18 @@ func evented(ec *EventCollector, page []types.BaseEvent) {
 			*types.VmMigratedEvent,
 			*types.DrsVmMigratedEvent,
 			*types.VmRelocatedEvent:
-			event = NewVMEvent(page[i])
-		}
 
-		if event != nil {
-			ec.callback(event)
+			// we have an event we need to process
+			ec.callback(NewVMEvent(page[i]))
+		case *types.VmReconfiguredEvent:
+			// reconfigures happen often, so completely ignore for now
+			return
+		default:
+			// log the skipped event
+			e := page[i].GetEvent()
+			log.Debugf("vSphere Event %s for eventID(%d) ignored by the event collector", e.FullFormattedMessage, int(e.Key))
 		}
```

Comment:
if you're concerned about duplicate events, should we store the EventID (assuming that's unique) or other identifying data just for the last page processed and check against that as a filter?.